src/client/src/setupTests.ts

```diff
@@ -0,0 +1,13 @@
+import { configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { IntlProvider, FormattedRelative } from "react-intl";
+
+/**
+ * This setup file configures the Enzyme Adapter and is executed before running the tests
+ * https://facebook.github.io/create-react-app/docs/running-tests#initializing-test-environment
+ */
+configure({ adapter: new Adapter() });
+
+const intlProvider = new IntlProvider({ locale: "en" }, {});
+const { intl } = intlProvider.getChildContext();
+global.intl = intl;
```

Comment:
Just for curiosity, how does integrating 'intl' in test configuration work/help?

Reply:
Some components use `intl` prop, I think it acts as a mock prop. You can try console.log and see the contents. @piotrci maybe you can help clarify?

Reply:
just to be clear, this was for handling components that use `react-intl`

https://github.com/formatjs/react-intl/blob/master/docs/Testing-with-React-Intl.md

Reply:
Yup. As @jsondoo said, thanks to that you can just set intl: global.intl in your tests.